catalog/core/catalog-core-metacardgroomerplugin/src/main/java/ddf/catalog/plugin/groomer/metacard/StandardMetacardGroomerPlugin.java

```diff
@@ -72,6 +73,10 @@ protected void applyCreatedOperationRules(CreateRequest createRequest, Metacard
             aMetacard.setAttribute(new AttributeImpl(Metacard.TAGS,
                     Collections.singletonList(Metacard.DEFAULT_TAG)));
         }
+
+        // upon creation of the metacard, generate attributes for created, modified dates
+        aMetacard.setAttribute(new AttributeImpl(ddf.catalog.data.types.Metacard.CREATED, now));
```

Comment:
Was gonna say the same thing. @bdeining, have you or @jlcsmith given this any thought?